lhotse/recipes/ami.py

```diff
@@ -13,31 +13,31 @@
 -- SDM: Single Distant Microphone
 -- MDM: Multiple Distant Microphones
 
-In this recipe, only the IHM setting is supported.
+In this recipe, IHM, IHM-Mix, and SDM settings are supported.
 """
 
 import logging
 import os
 import re
 import urllib.request
+import zipfile
 from collections import defaultdict
+import xml.etree.ElementTree as ET
 from gzip import GzipFile
 from pathlib import Path
-from typing import Dict, List, NamedTuple, Optional, Union
+from typing import Dict, List, NamedTuple, Optional, Union, Tuple
 
 # Workaround for SoundFile (torchaudio dep) raising exception when a native library, libsndfile1, is not installed.
 # Read-the-docs does not allow to modify the Docker containers used to build documentation...
 from tqdm.auto import tqdm
-
-if not os.environ.get('READTHEDOCS', False):
-    import torchaudio
+import soundfile as sf
 
 from lhotse.audio import AudioSource, Recording, RecordingSet
 from lhotse.supervision import SupervisionSegment, SupervisionSet
 from lhotse.utils import Pathlike, Seconds
 
-# TODO: support the "sdm" and "mdm" microphone settings
-mic = 'ihm'
+# TODO: support the "mdm" microphone setting. For "sdm" we currently use Array1-01.
+mics = ['ihm','ihm-mix','sdm']
```

Comment:
Could you show an example of 1-2 recordings from the `RecordingSet` manifest for the new setups? I remember that some time ago I changed the recipe to have all channels from a single session grouped as `AudioSource`s in a single `Recording` object, and I'm curious if the new setups also have that property. 

It might actually be nice to add such an example in the docstring of `prepare_ami` for illustration.

Reply:
(you can call `recording_set.to_yaml(path)` and paste one Recording - YAML is much easier to read than JSON)

Reply:
> Why don't you include `mdm`?

I'll add that soon.